models/migrations/v1_18/v230.go

```diff
@@ -10,8 +10,9 @@ import (
 // AddConfidentialColumnToOAuth2ApplicationTable: add ConfidentialClient column, setting existing rows to true
 func AddConfidentialClientColumnToOAuth2ApplicationTable(x *xorm.Engine) error {
 	type OAuth2Application struct {
+		ID                 int64
 		ConfidentialClient bool `xorm:"NOT NULL DEFAULT TRUE"`
 	}
 
-	return x.Sync(new(OAuth2Application))
+	return x.Table("oauth2_application").Sync(new(OAuth2Application))
```

Comment:
Does `Table` really work with `Sync`?

I think the could could be simplified to `type oauth2Application struct`, then no need to use TableName or Table method.

Reply:
99fef44